index/scorch/persister.go

```diff
@@ -996,6 +1066,45 @@ func (s *Scorch) removeOldZapFiles() error {
 	return nil
 }
 
+type snapshotMetaData struct {
+	epoch     uint64
+	timeStamp time.Time
+}
+
+func (s *Scorch) rootBoltSnapshotEpochTimeStamps() ([]*snapshotMetaData, error) {
```

Comment:
I'd suggest a better name, perhaps `rootBoltSnapshotMetaData()` owing to it's return values.

Reply:
Done.